### ISSUE TITLE
[mobile] Page from set a goal flow is automatically scrolled down like the previous one

### ISSUE THREAD

**Issue by viorelaioia** [mozilla/voice-web#1987]
**Env:** 
iPhone 7, iOS 12.2, Firefox and Safari
 
**STR:**

1. Login to CV on mobile
2. Navigate to Stats page 
3. Tap "Create a custom goal" on Speak section
4. Scroll down and tap "Set a goal" button

**Expected:**
Next page is shown from the beginning

**Actual:**
Next page is scrolled down, as the previous one. User needs to scroll up to see the first part of the page and to actually set the type of the personal goal. See [video](https://vimeo.com/333353297)

**Comment by f0cus10** [comment]
Is development even active on the iOS App? Last commits are from a year ago

**Comment by Gregoor** [comment]
Yes-ish, the iOS app is just a wrapper for the webview, which uses native APIs for recording (as Safari has not implemented them YET). So it gets "updated" anytime we update the site itself.

**Comment by MichaelKohler** [comment]
Is this still an issue on iOS? At least in the browser the page jumps to the top.

**Comment by moz-bozden** [comment]
Closing as stale...
